packages/lib-classifier/src/store/ClassificationStore.spec.js

```diff
@@ -124,13 +152,50 @@ describe('Model > ClassificationStore', function () {
         preventDefault: sinon.stub()
       }
       onComplete = sinon.stub()
-      rootStore.classifications.setOnComplete(onComplete)
-      rootStore.classifications.completeClassification(event)
+      classifications.setOnComplete(onComplete)
+    })
+    
+    beforeEach(function () {
+      classifications.createClassification(subject)
+      classifications.addAnnotation(0, { type: 'single', taskKey: 'T0' })
+      classifications.completeClassification(event)
+    })
+
+    afterEach(function () {
+      onComplete.resetHistory()
+      feedback.update.resetHistory()
+    })
+
+    after(function () {
+      feedback.createRules.restore()
+      feedback.update.restore()
+      feedback.reset.restore()
+    })
+
+    it('should update feedback', function () {
+      const annotation = {
+        task: 'T0',
+        value: 0
+      }
+      expect(feedback.update).to.have.been.calledOnceWith(annotation)
     })
 
     it('should call the onComplete callback with the classification and subject', function () {
-      const classification = rootStore.classifications.active
+      const classification = classifications.active
       expect(onComplete).to.have.been.calledOnceWith(classification.toJSON(), subject.toJSON())
     })
+
+    describe('classification metadata', function () {
+      let metadata
+
+      before(function () {
+        metadata = classifications.active.metadata
+      })
+
+      it('should have a feedback key', function () {
```

Comment:
I’m not completely happy with the wording of this test, but I wasn’t sure what `metadata.feedback` should be expected to be.